client/src/components/ManageArtist/ArtistForm.tsx

```diff
@@ -243,28 +260,47 @@ export const ArtistForm: React.FC<{
               {t("displayName")}:{" "}
               <InputEl {...register("name", { required: true })} />
             </FormComponent>
-            {existing && <ArtistFormColors />}
+            <div
```

Comment:
Could be a re-used component

Reply:
A styled-component looks like 

```
const Div = styled.div`
  margin-top: 2rem;
`
```

And then you're replace the next few lines with `<Div>`